File: QuickFiler/QuickFileControllerConstants.vb
Friend Module QuickFileControllerConstants

#Region "API Constants"
    Friend Const GWL_STYLE As Long = -16 'Sets a new window style  As LongPtr in 64bit version
    Friend Const WS_SYSMENU As Long = &H80000 'Windows style   As LongPtr in 64bit version
    Friend Const WS_THICKFRAME = &H40000 'Style that is apparently resizble
    Friend Const WS_MINIMIZEBOX As Long = &H20000   'As LongPtr in 64bit version
    Friend Const WS_MAXIMIZEBOX As Long = &H10000   'As LongPtr in 64bit version
    Friend Const SW_SHOWMAXIMIZED = 3
    Friend Const SW_FORCEMINIMIZE = 11
    Friend Const WM_SETFOCUS = &H7
    Friend Const GWL_HWNDPARENT = -8
    Friend Const olAppCLSN As String = "rctrl_renwnd32"
    Friend Const GA_PARENT = 1
    Friend Const GA_ROOTOWNER = 3
    Friend Const Modal = 0, Modeless = 1
#End Region

#Region "Form and Control Constants"
    Friend Const Top_Offset As Long = 6
    Friend Const Top_Offset_C As Long = 0
    Friend Const Left_frm As Long = 12
    Friend Const Left_lbl1 As Long = 6
    Friend Const Left_lbl2 As Long = 6
    Friend Const Left_lbl3 As Long = 6
    Friend Const Left_lbl4 As Long = 6
    Friend Const Left_lbl5 As Long = 372           'Folder:
    Friend Const Left_lblSender As Long = 66            '<SENDER>
    Friend Const Left_lblSender_C As Long = 6             '<SENDER> Compact view
    Friend Const Right_Aligned As Long = 648
    Friend Const Left_lblTriage As Long = 181           'X Triage placeholder
    Friend Const Left_lblActionable As Long = 198           '<ACTIONABL>
    Friend Const Left_lblSubject As Long = 66            '<SUBJECT>
    Friend Const Left_lblSubject_C As Long = 6             '<SUBJECT> Compact view
    Friend Const Left_lblBody As Long = 66            '<BODY>
    Friend Const Left_lblBody_C As Long = 6             '<BODY> Compact view
    Friend Const Left_lblSentOn As Long = 66            '<SENTON>
    Friend Const Left_lblSentOn_C As Long = 200           '<SENTON> Compact view
    Friend Const Left_lblConvCt As Long = 290           'Count of Conversation Members
    Friend Const Left_lblConvCt_C As Long = 320           'Count of Conversation Members Compact view
    Friend Const Left_lblPos As Long = 6             'ACCELERATOR Email Position
    Friend Const Left_cbxFolder As Long = 372           'Combo box containing Folder Suggestions
    Friend Const Left_inpt As Long = 408           'Input for folder search
    Friend Const Left_chbxGPConv As Long = 210           'Checkbox to Group Conversations
    Friend Const Left_chbxGPConv_C As Long = 372           'Checkbox to Group Conversations
    Friend Const Left_cbDelItem As Long = 588           'Delete email
    Friend Const Left_cbKllItem As Long = 618           'Remove mail from Processing
    Friend Const Left_cbFlagItem As Long = 569           'Flag as Task
    Friend Const Left_lblAcF As Long = 363           'ACCELERATOR F for Folder Search
    Friend Const Left_lblAcD As Long = 363           'ACCELERATOR D for Folder Dropdown
    Friend Const Left_lblAcC As Long = 384           'ACCELERATOR C for Grouping Conversations
    Friend Const Left_lblAcC_C As Long = 548           'ACCELERATOR C for Grouping Conversations
    Friend Const Left_lblAcX As Long = 594           'ACCELERATOR X for Delete email
    Friend Const Left_lblAcR As Long = 624           'ACCELERATOR R for remove item from list
    Friend Const Left_lblAcT As Long = 330           'ACCELERATOR T for Task ... Flag item and make it a task
    Friend Const Left_lblAcO As Long = 50            'ACCELERATOR O for Open Email
    Friend Const Left_lblAcO_C As Long = 0            'ACCELERATOR O for Open Email
    Friend Const Width_frm As Long = 655
    Friend Const Width_lbl1 As Long = 54
    Friend Const Width_lbl2 As Long = 54
    Friend Const Width_lbl3 As Long = 54
    Friend Const Width_lbl4 As Long = 52
    Friend Const Width_lbl5 As Long = 78            'Folder:
    Friend Const Width_lblSender As Long = 138           '<SENDER>
    Friend Const Width_lblSender_C As Long = 174           '<SENDER> Compact view
    Friend Const Width_lblTriage As Long = 11            'X Triage placeholder
    Friend Const Width_lblActionable As Long = 72            '<ACTIONABL>
    Friend Const Width_lblSubject As Long = 294           '<SUBJECT>
    Friend Const Width_lblSubject_C As Long = 354           '<SUBJECT> Compact view
    Friend Const Width_lblBody As Long = 294           '<BODY>
    Friend Const Width_lblBody_C As Long = 354           '<BODY> Compact view
    Friend Const Width_lblSentOn As Long = 80            '<SENTON>
    Friend Const Width_lblConvCt As Long = 30            'Count of Conversation Members
    Friend Const Width_lblPos As Long = 20            'ACCELERATOR Email Position
    Friend Const Width_cbxFolder As Long = 276           'Combo box containing Folder Suggestions
    Friend Const Width_inpt As Long = 156           'Input for folder search
    Friend Const Width_chbxGPConv As Long = 96            'Checkbox to Group Conversations
    Friend Const Width_cb As Long = 25            'Command buttons for: Delete email, Remove mail from Processing, and Flag as Task
    Friend Const Width_lblAc As Long = 14            'ACCELERATOR Width
    Friend Const Width_lblAcF As Long = 14            'ACCELERATOR F for Folder Search
    Friend Const Width_lblAcD As Long = 14            'ACCELERATOR D for Folder Dropdown
    Friend Const Width_lblAcC As Long = 14            'ACCELERATOR C for Grouping Conversations
    Friend Const Width_lblAcX As Long = 14            'ACCELERATOR X for Delete email
    Friend Const Width_lblAcR As Long = 14            'ACCELERATOR R for remove item from list
    Friend Const Width_lblAcT As Long = 14            'ACCELERATOR T for Task ... Flag item and make it a task
    Friend Const Width_lblAcO As Long = 14            'ACCELERATOR O for Open Email
    Friend Const Height_UserForm As Long = 149          'Minimum height of Userform
    Friend Const Width_UserForm As Long = 699.75        'Minimum width of Userform
    Friend Const Width_PanelMain As Long = 683           'Minimum width of _viewer.L1v1L2_PanelMain
    'Frame Design Constants
    Friend Const frmHt = 96
    Friend Const frmWd = 655
    Friend Const frmLt = 12
    Friend Const frmSp = 6
    Friend Const OK_left As Long = 216
    Friend Const CANCEL_left As Long = 354
    Friend Const OK_width As Long = 120
    Friend Const UNDO_left As Long = 480
    Friend Const UNDO_width As Long = 42
    Friend Const spn_left As Long = 606
#End Region

End Module

File: QuickFiler/QuickFileControllerConstants.cs

namespace QuickFiler
{
    internal static class QuickFileControllerConstants
    {

        #region API Constants
        internal const long GWL_STYLE = -16; // Sets a new window style  As LongPtr in 64bit version
        internal const long WS_SYSMENU = 0x80000L; // Windows style   As LongPtr in 64bit version
        internal const int WS_THICKFRAME = 0x40000; // Style that is apparently resizble
        internal const long WS_MINIMIZEBOX = 0x20000L;   // As LongPtr in 64bit version
        internal const long WS_MAXIMIZEBOX = 0x10000L;   // As LongPtr in 64bit version
        internal const int SW_SHOWMAXIMIZED = 3;
        internal const int SW_FORCEMINIMIZE = 11;
        internal const int WM_SETFOCUS = 0x7;
        internal const int GWL_HWNDPARENT = -8;
        internal const string olAppCLSN = "rctrl_renwnd32";
        internal const int GA_PARENT = 1;
        internal const int GA_ROOTOWNER = 3;
        internal const int Modal = 0;
        internal const int Modeless = 1;
        #endregion

        #region Form and Control Constants
        internal const long Top_Offset = 16L;
        internal const long Top_Offset_C = 0L;
        internal const long Left_frm = 12L;
        internal const long Left_lbl1 = 6L;
        internal const long Left_lbl2 = 6L;
        internal const long Left_lbl3 = 6L;
        internal const long Left_lbl4 = 6L;
        internal const long Left_lbl5 = 372L;           // Folder:
        internal const long Left_lblSender = 66L;            // <SENDER>
        internal const long Left_lblSender_C = 6L;             // <SENDER> Compact view
        internal const long Right_Aligned = 648L;
        internal const long Left_lblTriage = 181L;           // X Triage placeholder
        internal const long Left_lblActionable = 198L;           // <ACTIONABL>
        internal const long Left_lblSubject = 66L;            // <SUBJECT>
        internal const long Left_lblSubject_C = 6L;             // <SUBJECT> Compact view
        internal const long Left_lblBody = 66L;            // <BODY>
        internal const long Left_lblBody_C = 6L;             // <BODY> Compact view
        internal const long Left_lblSentOn = 66L;            // <SENTON>
        internal const long Left_lblSentOn_C = 200L;           // <SENTON> Compact view
        internal const long Left_lblConvCt = 290L;           // Count of Conversation Members
        internal const long Left_lblConvCt_C = 320L;           // Count of Conversation Members Compact view
        internal const long Left_lblPos = 6L;             // ACCELERATOR Email Position
        internal const long Left_cbxFolder = 372L;           // Combo box containing Folder Suggestions
        internal const long Left_inpt = 438L;           // Input for folder search 408 to 438
        internal const long Left_chbxGPConv = 210L;           // Checkbox to Group Conversations
        internal const long Left_chbxGPConv_C = 372L;           // Checkbox to Group Conversations
        internal const long Left_cbDelItem = 588L;           // Delete email
        internal const long Left_cbKllItem = 618L;           // Remove mail from Processing
        internal const long Left_cbFlagItem = 569L;           // Flag as Task
        internal const long Left_lblAcF = 363L;           // ACCELERATOR F for Folder Search
        internal const long Left_lblAcD = 363L;           // ACCELERATOR D for Folder Dropdown
        internal const long Left_lblAcC = 384L;           // ACCELERATOR C for Grouping Conversations
        internal const long Left_lblAcC_C = 548L;           // ACCELERATOR C for Grouping Conversations
        internal const long Left_lblAcX = 594L;           // ACCELERATOR X for Delete email
        internal const long Left_lblAcR = 624L;           // ACCELERATOR R for remove item from list
        internal const long Left_lblAcT = 330L;           // ACCELERATOR T for Task ... Flag item and make it a task
        internal const long Left_lblAcO = 50L;            // ACCELERATOR O for Open Email
        internal const long Left_lblAcO_C = 0L;            // ACCELERATOR O for Open Email
        internal const long Width_frm = 655L;
        internal const long Width_lbl1 = 54L;
        internal const long Width_lbl2 = 54L;
        internal const long Width_lbl3 = 54L;
        internal const long Width_lbl4 = 52L;
        internal const long Width_lbl5 = 78L;            // Folder:
        internal const long Width_lblSender = 138L;           // <SENDER>
        internal const long Width_lblSender_C = 174L;           // <SENDER> Compact view
        internal const long Width_lblTriage = 11L;            // X Triage placeholder
        internal const long Width_lblActionable = 72L;            // <ACTIONABL>
        internal const long Width_lblSubject = 294L;           // <SUBJECT>
        internal const long Width_lblSubject_C = 354L;           // <SUBJECT> Compact view
        internal const long Width_lblBody = 294L;           // <BODY>
        internal const long Width_lblBody_C = 354L;           // <BODY> Compact view
        internal const long Width_lblSentOn = 80L;            // <SENTON>
        internal const long Width_lblConvCt = 30L;            // Count of Conversation Members
        internal const long Width_lblPos = 20L;            // ACCELERATOR Email Position
        internal const long Width_cbxFolder = 276L;           // Combo box containing Folder Suggestions 
        internal const long Width_inpt = 126L;           // Input for folder search 156 to 126
        internal const long Width_chbxGPConv = 96L;            // Checkbox to Group Conversations
        internal const long Width_cb = 25L;            // Command buttons for: Delete email, Remove mail from Processing, and Flag as Task
        internal const long Width_lblAc = 14L;            // ACCELERATOR Width
        internal const long Width_lblAcF = 14L;            // ACCELERATOR F for Folder Search
        internal const long Width_lblAcD = 14L;            // ACCELERATOR D for Folder Dropdown
        internal const long Width_lblAcC = 14L;            // ACCELERATOR C for Grouping Conversations
        internal const long Width_lblAcX = 14L;            // ACCELERATOR X for Delete email
        internal const long Width_lblAcR = 14L;            // ACCELERATOR R for remove item from list
        internal const long Width_lblAcT = 14L;            // ACCELERATOR T for Task ... Flag item and make it a task
        internal const long Width_lblAcO = 14L;            // ACCELERATOR O for Open Email
        internal const long Height_UserForm = 149L;          // Minimum height of Userform
        internal const long Width_UserForm = 700L;        // Minimum width of Userform
        internal const long Width_PanelMain = 683L;           // Minimum width of _viewer.L1v1L2_PanelMain
                                                              // Frame Design Constants
        internal const int frmHt = 96;
        internal const int frmWd = 655;
        internal const int frmLt = 12;
        internal const int frmSp = 6;
        internal const long OK_left = 216L;
        internal const long CANCEL_left = 354L;
        internal const long OK_width = 120L;
        internal const long UNDO_left = 480L;
        internal const long UNDO_width = 42L;
        internal const long spn_left = 606L;
        #endregion

    }
}